src/_pytest/hookspec.py

```diff
@@ -158,7 +166,7 @@ def pytest_load_initial_conftests(early_config, parser, args):
 
 
 @hookspec(firstresult=True)
-def pytest_collection(session):
+def pytest_collection(session: "Session") -> Optional[Any]:
```

Comment:
`Optional[Any]`: the return values are not used, this is meant to indicate that.